student.rb

```diff
@@ -6,11 +6,12 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
+
```

Comment:
- [ ] Kindly add the setter`attr_reader :classroom` above the initialize method.

- [ ]  Instead of using `require`, kindly use `require_relative`. require_relative should be used for files while require should be used for gems. (Kindly make the changes to all other files where applicable)